### ISSUE TITLE
The Underwriters section needs to be reworked

### ISSUE THREAD

**Issue by BigActual** [Good-Shepherd-Catholic-Radio/Theme-2017#25]
Harry would like to use a rotator
I expressed to him the uselessness of rotators and that we could achieve his actual desire, which was to have this section take up less space. 

Our job is to analyze this space, determine a smaller form factor for Underwriters and resize the logos were the ones that we have can be. We then need to send that size to Harry so he can go to the underwriters and ask them for a new logo.

This could be a ratio locked shrinkage of the underwriter logos and then add a button to the side that says become an underwriter, which brings up that form. Just an idea.

**Comment by d4mation** [comment]
#19 ties in here.

**Comment by BigActual** [comment]
It could, but it's not a "must".

**Comment by d4mation** [comment]
We had talked about this on the phone at one point and I believe the conclusion we came to was that the different aspect ratios of each logo (and the inability to standardize them, given that they are logos) was a major limiting factor. As well as the fact that if they were to be made too small some of them would become illegible. 

Now that only 8 are being shown at once (Pending potential changes in #40, if they were to show all by clicking a button to slide down for example) we may have more wiggle room. However, if we were to do something like 1/3 width Become an Underwriter Form and 2/3 width Underwriters, we would still need to likely make the Underwriters have less per-row to ensure their logos are large enough to be read. This would mean either 2 per row, or we would need to raise or lower the number or Underwriters shown so that 3 could be shown per row without having a remainder straggling at the bottom.

We can give that a shot if you'd like. I think changing the number to 6 may be best so that we can have 2 rows of 3 and would allow it to visually balance with a header and button for the Become an Underwriter Form more easily.
